File: features/content/affiliate/src/main/java/com/tokopedia/affiliate/feature/createpost/data/pojo/Media.java

package com.tokopedia.affiliate.feature.createpost.data.pojo;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Media {

    @SerializedName("multiple_media")
    @Expose
    private boolean multipleMedia;
    @SerializedName("max_media")
    @Expose
    private int maxMedia;
    @SerializedName("media")
    @Expose
    private List<Medium> media = null;

    public boolean isMultipleMedia() {
        return multipleMedia;
    }

    public void setMultipleMedia(boolean multipleMedia) {
        this.multipleMedia = multipleMedia;
    }

    public int getMaxMedia() {
        return maxMedia;
    }

    public void setMaxMedia(int maxMedia) {
        this.maxMedia = maxMedia;
    }

    public List<Medium> getMedia() {
        return media;
    }

    public void setMedia(List<Medium> media) {
        this.media = media;
    }

}

File: features/content/kol_component/src/main/java/com/tokopedia/kolcomponent/data/pojo/feed/contentitem/Media.kt
package com.tokopedia.kolcomponent.data.pojo.feed.contentitem

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Media {
    @SerializedName("id")
    @Expose
    var id: String? = ""
    @SerializedName("type")
    @Expose
    var type: String? = ""
    @SerializedName("appLink")
    @Expose
    var appLink: String? = ""
    @SerializedName("webLink")
    @Expose
    var webLink: String? = ""
    @SerializedName("thumbnail")
    @Expose
    var thumbnail: String? = ""
    @SerializedName("ctaLink")
    @Expose
    var ctaLink: CtaLink? = CtaLink()
    @SerializedName("totalItems")
    @Expose
    var totalItems: Int = 0
    @SerializedName("isSelected")
    @Expose
    var isIsSelected: Boolean = false
    @SerializedName("mediaItems")
    @Expose
    var mediaItems: List<MediaItem>? = ArrayList()
}